Exercise_C/index.html

```diff
@@ -15,8 +15,19 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <div class = "container">
+      <div class = "box blue">
 
```

Comment:
Very smart to make two rows to ensure that your image is correct.
Well done Ivan